dev/io.openliberty.threading.virtual.internal/src/io/openliberty/threading/virtual/internal/VirtualThreadOperations.java

```diff
@@ -63,6 +82,9 @@ public boolean isVirtual(Thread thread) {
 
     @Override
     public boolean isSupported() {
-        return true;
+        if (overrideService != null)
+            return overrideService.allowVirtualThreadCreation();
+        else
+            return true;
```

Comment:
This method is supposed to be returning whether the Java level used supports virtual threads. This is still true regardless of which way the override to create them or not happens to be set at any given moment.
```suggestion
        return true;
```